build.sbt

```diff
@@ -1,8 +1,10 @@
-lazy val root = project.in(file("."))
+lazy val root = project
+  .in(file("."))
   .settings(
-    name        := "nsq-java-client",
-    version     := "1.2.0",
-    description := "Fast Java client for NSQ"
+    name := "nsq-java-client",
+    version := "1.3.0-rc1",
+    description := "Fast Java client for NSQ",
+    scalaVersion := "2.13.3"
```

Comment:
Do we even use scalac here? If we do then we cannot use it with out estate since we're mostly on 2.12, if we don't - what's the point?

Reply:
There is no .scala code (besides build files), so I removed scala version. I added `autoScalaLibrary := false`, as java code doesn't need scala library.